eucaconsole.init

```diff
@@ -66,8 +77,18 @@ start() {
         echo
     fi
 
+    # Ensure that /var/run/memcached has proper permissions
+    if [ "`stat -c %U /var/run/memcached`" != "$USER" ]; then
+        chown $USER /var/run/memcached
+    fi
```

Comment:
We don't own the system's copy of memcached.  If the person deploying the service goes out of his or her way to change the name of the user this runs as then it is fine to let him or her handle permissions as well.